### PR TITLE
addReviewForm

### PR DESCRIPTION
**Pull request by JineshMajithia-Bacancy**
add review form created with api and function.

### CODE REVIEW

File: src/Components/Forms/AddReviewForm.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            <br />
+            <FormGroup>
+                <Button type="submit" color="danger">Add Review</Button>
+            </FormGroup>
+        </Form>
+    )
+
+}
+
+export default AddReview;
```
Comment (harshvbacancy) [inline src/Components/Forms/AddReviewForm.tsx:48]: add extra

Comment (harshvbacancy) [inline src/Components/Forms/AddReviewForm.tsx:48]: add extra line at the end of the file

File: src/Components/Forms/AddProductForm.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            </FormGroup>
+            <br />
+            <FormGroup>
+                <Button color="primary" type="submit">Add Product</Button>
+            </FormGroup>
+        </Form>
+    )
+}
+
+export default AddProduct;
```
Comment (harshvbacancy) [inline src/Components/Forms/AddProductForm.tsx:63]: extra line

File: src/Components/Forms/AddProductForm.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+            userID: userId,
+            productTitle: (document.getElementById("title") as HTMLInputElement).value,
+            categoryID: (document.getElementById("category") as HTMLInputElement).value,
+            description: (document.getElementById("description") as HTMLInputElement).value,
+            reviews: []
+        }
+        axios.post("https://609cc6bd04bffa001792d455.mockapi.io/products", 
+        productData
+         ).then((response) => {
+            console.log(response)
```
Comment (harshvbacancy) [inline src/Components/Forms/AddProductForm.tsx:19]: remove all console log if it not necessary

File: src/Components/Forms/AddReviewForm.tsx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+}
+const AddReview: React.FC<Props> = (props: any) => {
+    
+    function onSubmit (event: any): void {
+        let productData;
+        let reviews = [];
+        let newReview = (document.getElementById("review") as HTMLInputElement).value
+        axios.get("https://609cc6bd04bffa001792d455.mockapi.io/products/"+props.productId,
+        ).then((response) => {
+            reviews = [response.data.reviews, newReview];
```
Comment (harshvbacancy) [inline src/Components/Forms/AddReviewForm.tsx:16]: we can use push method to add new review

**Review by harshvbacancy — CHANGES_REQUESTED** [review]

File: src/Components/Forms/AddReviewForm.tsx (hunk 4 above)
Comment (JineshMajithia-Bacancy) [reply src/Components/Forms/AddReviewForm.tsx]: we are pushing the whole product data whenever we update the reviews array. that is why we get the product, update the reviews and then use axios.put to update the object.

File: src/Components/Forms/AddProductForm.tsx (hunk 3 above)
Comment (JineshMajithia-Bacancy) [reply src/Components/Forms/AddProductForm.tsx]: they are just to log the response. if we do not have the need to do anything with the response, i will remove .then and .catch.

**Comment by rohanBacancy** [comment]
Resolve the conflict

**Pull request merged**